### PR TITLE
Follow up changes to Runic formatting

### PR DESCRIPTION
**Pull request by fredrikekre**
- Add commit to blame ignore list
 - Mention Runic formatting in the CHANGELOG.md
 - Run Runic check on CI

### CODE REVIEW

**Comment by KnutAM** [comment]
Really nice with having Runic here! 

I think some instructions for how to run locally (both as git hook and without) in CONTRIUBTING would be helpful. 
Does it also make sense to add some extra config / scripts to facilitate that process?

**Comment by fredrikekre** [comment]
Added some info to CONTRIBUTING.md, perhaps you can try it out?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,6 +70,35 @@ issues][open-issues].
 Remember to always include (when applicable): i) unit tests which exercises the new code,
 ii) documentation, iii) a note in the [CHANGELOG.md](CHANGELOG.md) file.
 
+### Code formatting
+
+Ferrite uses [Runic.jl][runic] for code formatting. You can install Runic with the following
+command:
+
+```shell
+julia --project=@runic -e 'using Pkg; Pkg.add("Runic")'
```
Comment (KnutAM) [inline CONTRIBUTING.md:79]:
```suggestion
julia --project=@runic -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl")'
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -70,6 +70,35 @@ issues][open-issues].
...
+
+Ferrite uses [pre-commit] to run various checks (including Runic formatting). After
+installing pre-commit (refer to the [pre-commit installation
+instructions][pre-commit-install]) run the following from the root of the repository to
+configure pre-commit to run before each commit:
+
+```shell
+pre-commit install
+```
+
```
Comment (KnutAM) [inline CONTRIBUTING.md:101]: Not sure if this is completely accurate, but I assume that it is at least a required step? I installed first anyway, and didn't test without installing first...

```suggestion
Note that this requires `Runic.jl` to be installed in the shared `@runic` environment according to the instructions above
```

**Review by KnutAM — COMMENTED** [review]
Works for me with these suggested changes

**Pull request merged**